lib/model/survey_question_model.dart

```diff
@@ -35,6 +36,7 @@ class SurveyQuestionModel extends Equatable {
     required this.coverImageUrl,
     required this.largeCoverImageUrl,
     required this.answers,
+    required this.totalQuestions,
```

Comment:
I think the `totalQuestions` should not be the question's property, with the list of question, it will redundant IMO. Why don't we just have it as the param on the question screen instead? 🤔

Reply:
https://github.com/Wadeewee/survey-flutter-ic-pooh/pull/56#discussion_r1211047177

If we follow this way ☝️ I think we don't need this field.